src/aat/java/uk/gov/hmcts/reform/em/annotation/functional/RectangleScenarios.java

```diff
@@ -0,0 +1,387 @@
+    private String testUrl;
+
+    @Rule
+    public RetryRule retryRule = new RetryRule(3);
+
+    private RequestSpecification request;
+    private RequestSpecification unAuthenticatedRequest;
+
+    @Before
+    public void setupRequestSpecification() {
+        request = testUtil
+                .authRequest()
+                .baseUri(testUrl)
+                .contentType(APPLICATION_JSON_VALUE);
+
+        unAuthenticatedRequest = testUtil
+                .unauthenticatedRequest()
+                .baseUri(testUrl)
+                .contentType(APPLICATION_JSON_VALUE);
+    }
+
+    @Test
+    public void shouldReturn201WhenCreateNewRectangle() {
+        final String newAnnotationSetId = createAnnotationSet();
+        final String annotationId = createAnnotation(newAnnotationSetId);
+        final String rectangleId = UUID.randomUUID().toString();
+
+        final ValidatableResponse response = createRectangle(annotationId, rectangleId);
+
+        response
+                .statusCode(201)
+                .body("x", equalTo(1f))
+                .body("y", equalTo(2f))
+                .body("width", equalTo(10f))
+                .body("height", equalTo(11f))
+                .body("annotationId", equalTo(annotationId))
+                .header("Location", equalTo("/api/rectangles/" + rectangleId))
+                .log().all();
+    }
+
+    @Test
+    public void shouldReturn400WhenCreateNewRectangle() {
+        final String newAnnotationSetId = createAnnotationSet();
+        final String annotationId = createAnnotation(newAnnotationSetId);
+        final String rectangleId = UUID.randomUUID().toString();
+        final JSONObject rectanglePayload = createRectanglePayload(annotationId, rectangleId);
+
+        rectanglePayload.remove("id");
+
+        request
+                .body(rectanglePayload.toString())
+                .post("/api/rectangles")
+                .then()
+                .statusCode(400)
+                .log().all();
+    }
+
+    @Test
+    public void shouldReturn401WhenCreateNewRectangle() {
+        final String annotationId = UUID.randomUUID().toString();
+        final String rectangleId = UUID.randomUUID().toString();
+        final JSONObject rectanglePayload = createRectanglePayload(annotationId, rectangleId);
+
+        unAuthenticatedRequest
+                .body(rectanglePayload.toString())
+                .post("/api/rectangles")
+                .then()
+                .statusCode(401)
+                .log().all();
+    }
+
+    @Test
+    public void shouldReturn500WhenCreateNewRectangle() {
+        final String annotationId = UUID.randomUUID().toString();
+        final String rectangleId = UUID.randomUUID().toString();
+        final JSONObject rectanglePayload = createRectanglePayload(annotationId, rectangleId);
+        request
+                .body(rectanglePayload.toString())
+                .post("/api/rectangles")
+                .then()
+                .statusCode(500)
```

Comment:
why?

Reply:
its same as above issue, raised a bug for this issue as well.